sdk/servicebus/azure-servicebus/tests/perf_tests/T1_legacy_tests/_test_base.py

```diff
@@ -107,24 +116,19 @@ def __init__(self, arguments):
             mode=mode,
             prefetch=self.args.num_messages,
             idle_timeout=self.args.max_wait_time)
-    
+        self.receiver.open()
+        await self.async_receiver.open()
+        await self._preload_queue()
```

Comment:
The call to `_preload_queue` moved from `global_setup` to instance `setup`.  Intentional or mistake?

Reply:
You're right - this should be global